docs/en/serving.md

```diff
@@ -76,7 +76,7 @@ python3 -m fastchat.model.apply_delta \
   --target-model-path /path/to/vicuna-13b \
   --delta-path lmsys/vicuna-13b-delta-v1.1
 
-python3 lmdeploy/serve/turbomind/deploy.py vicuna-13B /path/to/vicuna-13b hf
+python3 -m lmdeploy/serve/turbomind/deploy.py vicuna-13B /path/to/vicuna-13b hf
```

Comment:
may change 'lmdeploy/serve/turbomind/deploy.py' to 'lmdeploy.serve.turbomind.deploy'